tests/base/src/test/java/org/wildfly/security/http/impl/AbstractBaseHttpTest.java

```diff
@@ -459,23 +464,40 @@ protected CallbackHandler getCallbackHandler(String username, String realm, Stri
                         throw new IllegalStateException(e);
                     }
                 } else if (callback instanceof EvidenceVerifyCallback) {
-                    PasswordGuessEvidence evidence = (PasswordGuessEvidence) ((EvidenceVerifyCallback) callback).getEvidence();
-                    ((EvidenceVerifyCallback) callback).setVerified(Arrays.equals(evidence.getGuess(), password.toCharArray()));
-                    evidence.destroy();
+                    if(((EvidenceVerifyCallback) callback).getEvidence() instanceof PasswordGuessEvidence){
```

Comment:
@keshav-redhat Just a minor, notice that in the rest of the code, we use spaces around `if` and `else`. So instead of eg. `}else if(` it should be `} else if (` . This could be fixed here and below